about.html

```diff
@@ -13,52 +13,88 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
-
-    <img src="img/about-plan.png" alt="strategy">
+        <header class="head-cont-about">
+            <Img class="logo-top-about" src="./img/lambda-black.png"></Img>
     
```

Comment:
the image tag should be lowercase here instead of Img and you can omit the ./ in the src attribute and still access it, you would also want to keep an alt tag on all your images as screen readers will have issues and if the image doesnt load for some reason, there will be no info for the user other than that little exed out box, with an alt tag it will at least display the description you give it.